### PR TITLE
Add mayhem support

### PR DESCRIPTION
**Pull request by WGibbs15**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+project: wgibbs15/capstone-rs-mayhem
```
Comment (xansec) [inline Mayhemfile:1]: Please use generic project names.

File: capstone-rs/fuzz/fuzz_targets/fuzz_target_disasm_x86_64.rs [hunk 2]
```diff
@@ -15,8 +15,8 @@ fuzz_target!(|data: &[u8]| {
         let detail: InsnDetail = cs.insn_detail(&i).unwrap();
```
Comment (xansec) [inline capstone-rs/fuzz/fuzz_targets/fuzz_target_disasm_x86_64.rs:15]: Can you explain this change?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: capstone-rs/fuzz/fuzz_targets/fuzz_target_disasm_x86_64.rs (hunk 2 above)
Comment (WGibbs15) [reply capstone-rs/fuzz/fuzz_targets/fuzz_target_disasm_x86_64.rs:15]: This file wasn't touched for years, but Capstone updated to a newer version and changed some function return values.

Comment (xansec) [reply capstone-rs/fuzz/fuzz_targets/fuzz_target_disasm_x86_64.rs:15]: Gotcha, thanks!

**Pull request merged**